### ISSUE TITLE
Gradle Dependencie

### ISSUE THREAD

**Issue by ashenriques** [RichRelevance/rr-android-sdk#3]
I would like to have access to RichRelevance through a Gradle dependency.
I don't understand why in iOS you have it as a Cocoa Pod, but in Android you have to setup manually.

**Comment by mhswtf** [comment]
I agree with this. It should be the default for any android sdk to have a gradle dependency. Is there any plans to create one?

**Comment by ramav89** [comment]
Same here, i feel it is too much to do as .aar file and locally use it. it may cause for us to miss the updates.
and also it is a disadvantage for dynamic feature
